src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,95 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
+    private T[] elements;
+    private int size;
+
+    public ArrayList() {
+
+        elements = (T[]) new Object[CAPACITY];
+    }
+
     @Override
     public void add(T value) {
-
+        if (size == elements.length) {
+            grow();
+        }
+        elements[size++] = value;
```

Comment:
```suggestion
        add(value, size);
```

Reply:
next time let's try to explain why the changes you suggest are better, because it is not always obvious